guides/v2.1/cloud/env/variables-global.md

```diff
@@ -0,0 +1,72 @@
+---
+group: cloud-guide
+title: Global variables
+functional_areas:
+  - Cloud
+  - Configuration
+---
+
+The following _global_ variables control actions in the build, deploy, and post-deploy stages of the `.magento.env.yaml` file. Because global variables impact every stage, you must set them in the `global` stage. Insert these variables in the `global` stage of the `.magento.env.yaml` file:
+
+```yaml
+stage:
+  global:
+    GLOBAL_VARIABLE_NAME: value
+```
+### `MIN_LOGGING_LEVEL`
+
+-  **Default**—_Not set_
+-  **Version**—Magento 2.1.4 and later
+
+Overrides the minimum logging level for all output streams without making changes to the code. This helps to improve troubleshooting problems with deployment. For example, if your deployment fails, you can use this variable to increase the logging granularity globally. See [Set up notifications—Log levels]({{ page.baseurl }}/cloud/env/setup-notifications.html#log-levels). The `min_level` value in Logging handlers overwrites this setting.
+
+```yaml
+stage:
+    global:
+        MIN_LOGGING_LEVEL: debug
+```
+### `SCD_ON_DEMAND`
+
+-  **Default**—_Not set_
+-  **Version**—Magento 2.1.4 and later
+
+Enable generation of static content when requested by a user. This is ideal for the development and testing workflow, because it decreases the deployment time.
+
+Pre-loading the cache using the [`post_deploy` hook]({{ page.baseurl }}/cloud/project/project-conf-files_magento-app.html#hooks) reduces site downtime. The cache warming is available only for Pro projects that contain Staging and Production environments in the Project Web UI and for Starter projects. Add the `SCD_ON_DEMAND` environment variable to the `global` stage in the `.magento.env.yaml` file:
```

Comment:
I'm not sure of the connection between the first paragraph and the second.  I know the SCD concepts are complex. Not sure if there is a way to connect the concepts better -- pre-loading the cache  relationship to SCD_ON_DEMAND.